examples/chip-tool/commands/discover/DiscoverCommissionablesCommand.h

```diff
@@ -20,6 +20,8 @@
 
 #include "../common/CHIPCommand.h"
 
+#include <lib/dnssd/Advertiser.h>
```

Comment:
Where does this file use Advertiser?  Seems like it should just include `src/lib/dnssd/Types.h`, no?

Reply:
No, it's not:

```
 % git grep DiscoveryFilterType | grep enum | grep h:
src/lib/dnssd/Types.h:enum class DiscoveryFilterType : uint8_t
```

Reply:
To clarify, I meant "enum class CommissioningMode", which is defined in ./src/lib/dnssd/Advertiser.h.